projects/sound-open-firmware/build.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash -eux
+
+cd $SRC/sof/tools/oss-fuzz
+rm -rf build_oss_fuzz
+mkdir -p build_oss_fuzz
+cd build_oss_fuzz
+
+export VERBOSE=1
+cmake -DCMAKE_INSTALL_PREFIX=install -DCMAKE_LINKER=$CXX -DCMAKE_C_LINK_EXECUTABLE="<CMAKE_LINKER> <FLAGS> <CMAKE_C_LINK_FLAGS> <LINK_FLAGS> <OBJECTS> -o <TARGET> <LINK_LIBRARIES>" ..
```

Comment:
why are these values from filled?
<CMAKE_LINKER> <FLAGS> <CMAKE_C_LINK_FLAGS> <LINK_FLAGS> <OBJECTS> -o <TARGET> <LINK_LIBRARIES>

Reply:
Do you mean why or where?

Where) Cmake fills them as part of the process
Why) Cmake for some reason refuses to use CXX without the arguements.

Reply:
ok fine then.